src/snowflake/snowpark/dataframe.py

```diff
@@ -707,24 +711,51 @@ def select(
 
             >>> df_selected = df.select(df["col1"], df.col2, df.col("col3"))
 
+        Example 5::
+
+            >>> from snowflake.snowpark.functions import table_function
+            >>> split_to_table = table_function("split_to_table")
+            >>> df_selected = df.select(df.col1, split_to_table(df.col2, lit(" ")), df.col("col3"))
+
         Args:
-            *cols: A :class:`Column`, :class:`str`, or a list of those.
+            *cols: A :class:`Column`, :class:`str`, :class:`TableFunctionCall`, or a list of those. Note that at most one
+                   :class:`TableFunctionCall` object is supported within a select statement.
```

Comment:
```suggestion
                   :class:`TableFunctionCall` object is supported within a select call.
```
nit: I would avoid "statement" since it's not in the SQL